mindone/diffusers/pipelines/stable_diffusion_xl/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .pipeline_output import StableDiffusionXLPipelineOutput
+from .pipeline_stable_diffusion_inpainting import StableDiffusionXLInpaintPipeline
```

Comment:
It should be `pipeline_stable_diffusion_xl_inpaint`, right?

Reply:
Yes, changed it